AdminWebsite/AdminWebsite/ClientApp/src/app/get-audio-file/get-audio-link-button/get-audio-link-button.component.html

```diff
@@ -15,7 +15,7 @@
 </div>
 
 <div class="button-div" *ngIf="showOnState(audioLinkStates.finished)">
-  <div *ngFor="let audioLink of audioLinks; let fileIndex = index">
+  <div *ngFor="let audioLink of getAudioLinks(); let fileIndex = index">
```

Comment:
Was this change intentional? I know it was mentioned about the getter but I thought that was in get-audio-file-component